### DISCUSSION TITLE
When I updated to the latest version of nuxt3, I got an error "[h3] [unhandled] Parse Error: Header overflow"

### DISCUSSION THREAD

**Discussion by HochinChan** [nuxt/nuxt#16561] [Questions]
![image](https://user-images.githubusercontent.com/65072526/196080303-85264cc9-b286-4c96-bcc3-80bfa976c009.png)
This error causes all the requests on my page to get a 500 status code.
![image](https://user-images.githubusercontent.com/65072526/196080422-9657a24c-effb-4838-ba49-ae297f852a22.png)

Does anyone know how to fix this error?

**Comment by HochinChan** [comment]
👀👀👀👀

**Comment by TheAlexLichter** [comment]
Hey! Could you kindly provide a [reproduction](https://v3.nuxtjs.org/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏

**Reply by HochinChan** [reply]
I found that the root of the problem doesn't seem to be this error, but the vite proxy not taking effect.

**Reply by HochinChan** [reply]
Can you help me see why useFetch is not being proxied by vite?
https://stackblitz.com/edit/nuxt-starter-2ohy1y?file=pages/index.vue

**Reply by HochinChan** [reply]
![image](https://user-images.githubusercontent.com/65072526/197456946-eaef05f1-636f-4126-a594-6bd84b919ff9.png)
The console prints this when I send the request, but it's not what I want, I need to proxy the request to another server

**Reply by HochinChan** [reply]
This is working fine in rc11
